pycytominer/cyto_utils/output.py

```diff
@@ -2,19 +2,21 @@
 Utility function to compress output data
 """
 
-import os
-import warnings
+from typing import Dict, Union, Optional
+
 import pandas as pd
 
-compress_options = ["gzip", None]
+COMPRESS_OPTIONS = ["gzip", None]
 
 
 def output(
-    df,
-    output_filename,
-    sep=",",
-    float_format=None,
-    compression_options={"method": "gzip", "mtime": 1},
+    df: pd.DataFrame,
+    output_filename: str,
+    output_type: str = "csv",
+    sep: str = ",",
+    float_format: Optional[str] = None,
+    compression_options: Union[str, Dict] = {"method": "gzip", "mtime": 1},
+    **kwargs,
```

Comment:
Why is adding `kwargs` here necessary? Am I missing something?

Reply:
Apologies, I originally intended to use `kwargs` for `to_parquet` parameter handling but found that the related PyArrow functions are unable to flex to this and ended up removing. I'll remove `kwargs` as appropriate here.

Reply:
A quick follow up and correction here: `**kwargs` are used in this instance to enable `merge_single_cells` to utilize both annotation and output arguments together (and not conflict when seeing additional unrecognized arguments).